### ISSUE TITLE
Teamspeak containter using NFS share

### ISSUE THREAD

**Issue by BoxedBrain** [TeamSpeak-Systems/teamspeak-linux-docker-images#17]
Hello, I could need some help here.
I try to use a NFS volume for persistant data and the server won't boot.  If I use a local storage, the server start just instantly.

&#x200B;

    2019-02-04 14:26:58.835414|INFO    |ServerLibPriv |   |TeamSpeak 3 Server 3.6.1 (2019-01-30 07:34:52)
    2019-02-04 14:26:58.836668|INFO    |ServerLibPriv |   |SystemInformation: Linux 4.15.0-43-generic #46-Ubuntu SMP Thu Dec 6 14:45:28 UTC 2018 x86_64 Binary: 64bit
    2019-02-04 14:26:58.836728|INFO    |ServerLibPriv |   |Using hardware aes
    2019-02-04 14:26:58.837167|INFO    |DatabaseQuery |   |dbPlugin name:    SQLite3 plugin, Version 3, (c)TeamSpeak Systems GmbH
    2019-02-04 14:26:58.837512|INFO    |DatabaseQuery |   |dbPlugin version: 3.11.1
    2019-02-04 14:26:58.838788|INFO    |DatabaseQuery |   |checking database integrity (may take a while)

My NFS Volume is created like this:

    docker volume create --driver local --opt type=nfs \
        --opt o=addr=<IP>,rw --opt device=:/<export>/swarm/TeamspeakData --name=TeamspeakData

Service started like this:

    docker service create \
        --replicas 1 --name Teamspeak -e TS3SERVER_LICENSE=accept -p 9987:9987/udp -p 10011:10011 -p 30033:30033 --network skynet \
        --mount "type=volume,source=TeamspeakData,destination=/var/ts3server,readonly=false" teamspeak

Thanks for any advice on this!

**Comment by muenchow** [comment]
Is the log from a failed server start up?
Did you try using mariadb instead of sqlite?

**Comment by BoxedBrain** [comment]
Yes, its my original output. I did not tried mariadb yet because I later want to import my backup sqlite.db
